example/lib/main.dart

```diff
@@ -64,7 +64,7 @@ class _MyAppState extends State<MyApp> {
   Future<void> initData() async {
     // Initialize repository
     await initRepository();
-    final CoreStore coreStore = await initCoreStore();
+    // final CoreStore coreStore = await initCoreStore();
```

Comment:
Why has this been removed?

Reply:
bwcause it is not used yet. so I comment it out.

Reply:
This seems like a big and will need to be investigated further.